src/tests/setup.ts

```diff
@@ -1,3 +1,4 @@
 import { vi } from 'vitest';
+import '@testing-library/jest-dom/vitest';
```

Comment:
Had to add this to be able to use `toBeInDocument()`/`toHaveFocus()` etc. Not quite sure if this is the correct place though.